src/css/editable.less

```diff
@@ -5,7 +5,6 @@
     width: 100%;
     padding: 2px;
     .box-sizing(border-box); // supported by IE8+: http://caniuse.com/css3-boxsizing
-    white-space: pre;
```

Comment:
`white-space` defaults to `normal` which causes things to wrap:
![image](https://cloud.githubusercontent.com/assets/225809/7972515/fdaa6262-0a01-11e5-8759-923b5d2b4855.png)
but if we change this line to `white-space: nowrap` instead, with the `&nbsp;` change, I can't think of any further problems.